publish-service/src/main/resources/application.properties

```diff
@@ -36,3 +37,7 @@ activedirectory.managerDn:
 activedirectory.rootDn:
 activedirectory.userSearchFilter:
 activedirectory.connectionTimeOut:
+
+spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
```

Comment:
Do we have Glue ticket adding this property?

Reply:
This is necessary due to uplift of spring-boot. Nothing to configure by user.